app/serializers/api/v4/run_histories_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class Api::V4::RunHistoriesSerializer < Api::V4::ApplicationSerializer
```

Comment:
Keeping with the pattern of other serializers, this seems like it should be `Api::V4::RunHistorySerializer` (a "history" being a single entry).

Reply:
Not really sure what I was thinking with the naming on these, will change to non-plural.